package/MDAnalysis/analysis/align.py

```diff
@@ -359,29 +424,175 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     ref_coordinates = ref_atoms.positions - ref_com
     mobile_coordinates = mobile_atoms.positions - mobile_com
 
-    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates)
-
-    R, new_rmsd = rotation_matrix(mobile_coordinates, ref_coordinates, weights=weights)
+    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates, weights)
 
     if subselection is None:
-        atoms = mobile.universe.atoms
-    elif type(subselection) is str:
-        atoms = mobile.select_atoms(subselection)
+        # mobile_atoms is Universe
+        mobile_atoms = mobile.universe.atoms
+    elif isinstance(subselection, str):
+        # select mobile_atoms from string
+        mobile_atoms = mobile.select_atoms(subselection)
     else:
         try:
-            atoms = subselection.atoms
+            # treat subselection as AtomGroup
+            mobile_atoms = subselection.atoms
         except AttributeError:
-            raise TypeError("subselection must be a selection string, a AtomGroup or Universe or None")
-
-    atoms.translate(-mobile_com)
-    atoms.rotate(R)
-    atoms.translate(ref_com)
+            raise TypeError("subselection must be a selection string, an"
+                            " AtomGroup or Universe or None")
 
+    # _fit_to DOES subtract center of mass, will provide proper min_rmsd
+    mobile_atoms, new_rmsd = _fit_to(mobile_coordinates, ref_coordinates,
+                                     mobile_atoms, mobile_com, ref_com,
+                                     weights=weights)
     return old_rmsd, new_rmsd
 
 
-def rms_fit_trj(traj, reference, select='all', filename=None, rmsdfile=None, prefix='rmsfit_',
-                mass_weighted=False, tol_mass=0.1, strict=False, force=True, quiet=False, **kwargs):
+class AlignTraj(AnalysisBase):
+    """rms_align trajectory to a reference structure using a selection.
```

Comment:
No but all public variables like `self.rmsd` or `self.writer`. Important is here `self.rmsd` since it contains valuable information but it is nowhere documented in this class.  I know I'm pedantic here with the documentation but I think it is good if we start to document variables that are calculated during an analysis.

We shouldn't document private variables. 'self.writer` could be private with `self._writer`. This variable is an implementation detail and doesn't contain valuable information for the user. These private variables which are implementation details don't need to be documented.